nautiluszim/scraper.py

```diff
@@ -156,6 +156,10 @@ def files_path(self):
 
     def run(self):
         """ execute the scrapper step by step """
+
+        # fail early if description length is greater than 80 characters
```

Comment:
I have ensured that the check for description length takes place in the beginning itself.

Reply:
We don't need this comment